fiaas_deploy_daemon/config.py

```diff
@@ -328,6 +328,15 @@ def _parse_args(self, args):
             default="1",
             type=_int_or_unicode
         )
+        parser.add_argument(
+            "--disable-service-links",
+            dest="enable_service_links",
```

Comment:
For me, this is confusing 😅.
If the flag is --disable-service-links, the config should be the same 😄

Reply:
I agree, but I think this is the reasonable way of doing it due to some unfortunate circumstances.

1) The kubernetes API takes the argument as enableServiceLinks, with default for now being true.
2) The library used for parsing command line arguments doesn't take an argument for boolean arguments, so it's not possible to send in false together with it. My initial attempt was with the argument as "enable-service-links", but hit this problem. 

You do definitely have a point that it's confusing however.

Reply:
Maybe just add a comment explaining this?

Reply:
@henrik242 I've added a comment about the inverted logic.